src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -320,7 +321,8 @@ public function getAttributes()
     {
         return [
             'data-editor' => 'tinyMCE', // Register ss.editorWrappers.tinyMCE
-            'data-config' => Convert::array2json($this->getConfig())
+            'data-config' => Convert::array2json($this->getConfig()),
+            'style' => 'height:350px'
```

Comment:
Oh, or maybe remove `min_height` setting and keep the `style` :)
I don't believe both is needed, either way.